src/test/java/be/syntra/devshop/DevshopFront/controllers/SearchControllerTest.java

```diff
@@ -46,203 +42,180 @@
+        final CartDto dummyCart = getCartWithMultipleNonArchivedProducts();
+        when(productService.findAllProductsBySearchModel()).thenReturn(dummyProductListDto);
+        when(productMapperUtil.convertToProductDtoList(any())).thenReturn(getDummyProductDtoList());
+        when(searchService.getSearchModel()).thenReturn(new SearchModel());
         when(cartService.getCart()).thenReturn(dummyCart);
-        when(searchService.getSearchModel()).thenReturn(dummySearchModel);
 
         // when
-        final ResultActions getResult = mockMvc.perform(get("/search/?searchRequest=" + searchRequest));
+        final ResultActions getResult = mockMvc.perform(get("/search/?searchRequest=" + TEST_REQUEST));
 
         // then
         getResult
                 .andExpect(status().isOk())
                 .andExpect(view().name("product/productOverview"))
                 .andExpect(content().contentType("text/html;charset=UTF-8"))
-                .andExpect(model().attributeExists("products"))
-                .andExpect(model().attribute("products", dummyProducts))
-                .andExpect(model().attribute("cart", dummyCart));
+                .andExpect(model().attributeExists("searchModel", "productlist", "cart"));
 
-        verify(productListCacheService, times(1)).findBySearchRequest(any());
-        verify(productListCacheService, times(1)).filterByPrice(any(), any());
+        verify(searchService, times(1)).setSearchRequest(TEST_REQUEST);
+        verify(searchService, times(1)).setSearchResultView(true);
+        verify(searchService, times(1)).setArchivedView(false);
+        verify(productService, times(1)).findAllProductsBySearchModel();
     }
 
     @Test
-    void canSearchForDescriptionTest() throws Exception {
-        // given
-        final String searchRequest = "product";
-        final String description = "another";
-        final List<Product> dummyProducts = getDummyNonArchivedProductList();
-        final ProductList dummyProductList = new ProductList(dummyProducts);
-        final CartDto dummyCart = getCartWithOneDummyProduct();
-        final SearchModel dummySearchModel = new SearchModel();
-        BigDecimal priceLow = new BigDecimal("0");
-        BigDecimal priceHigh = new BigDecimal("10000");
-        dummySearchModel.setPriceLow(priceLow);
-        dummySearchModel.setPriceHigh(priceHigh);
-        dummySearchModel.setDescription(description);
-        dummySearchModel.setSearchRequest(searchRequest);
-
-        when(productListCacheService.findBySearchRequest(any())).thenReturn(dummyProductList);
-        when(productListCacheService.filterByPrice(dummyProducts, dummySearchModel)).thenReturn(dummyProductList);
-        when(productListCacheService.searchForProductDescription(dummyProducts, dummySearchModel)).thenReturn(dummyProductList);
+    void searchPriceLowTest() throws Exception {
+        //given
+        final String PRICE_LOW = "6.66";
```

Comment:
not a static String. Convention-wise, it shouldn't be allCaps

Reply:
idd, changed